stories/troubleshooting.stories.mdx

```diff
@@ -0,0 +1,59 @@
+<Meta title="Documentation/Troubleshooting"/>
+
+# Troubleshooting
+Looking to get involved in the mono repo but running into issues? Read on to find out how to fix them!
+
+## Getting started
+
+### First time setup
+Every time you get the latest changes in the mono repository, you need to follow a few short steps to get up and running
+
+1. Install yarn packages at the root: `yarn`
+2. Run the build command at the root: `yarn build`
+3. You can now run storybook using the command `yarn storybook:serve`
+
+### Cannot find module `@justeat/f-<any-package-name>`
+This issue indicates that you have not run yarn build with the latest code, refer to `First time setup` for a solution.
+
+### Storybook error reading: `ENOENT: no such file or directory, open ...`
+This issue indicates that you have not run yarn build with the latest code, refer to `First time setup` for a solution.
+
+## Development Process
+
+### I am unable to run the 'demo' script within a component
+- We recently changed the way interaction based tests are executed; they no longer depend on a demo script, but are now run within storybook
+- As a result of those changes, the demo script you may have previously used has been removed
+- If you find you have a use case where you need to demo a component outside of storybook; contact the team for assistance.
+
+### How can I test an unpublished version of my component in another project?
+Sometimes we might want to test changes to a component before we submit a PR, you can use yalc to do this.
+
+- Read more about yalc and find installation instructions here - https://github.com/whitecolor/yalc
```

Comment:
I'd update to this to avoid having to have the link as text:

`Read more [about yalc and how to install it](https://github.com/whitecolor/yalc).`